native/src/components/CityNotCooperatingFooter.tsx

```diff
@@ -50,7 +50,7 @@ const CityNotCooperatingFooter = ({
     <FooterContainer>
       <StyledIcon Icon={CityNotCooperatingIcon} />
       <Question>{t('cityNotFound')}</Question>
-      <StyledButton text={t('clickHere')} onPress={navigateToCityNotCooperating} />
+      <StyledButton text={t('clickHere', { appName: buildConfig().appName })} onPress={navigateToCityNotCooperating} />
```

Comment:
:wrench: It would be cool if you could change the translation key as well, e.g. to `suggestToRegion` or similar. Same for other occurrences.